examples/keywords/test/test.ts

```diff
@@ -0,0 +1,170 @@
+import { Principal } from 'azle';
+import { run_tests, Test } from 'azle/test';
+import { execSync } from 'child_process';
+import { createActor } from './dfx_generated/keywords';
+import {
+    KeywordRecord,
+    KeywordVariant,
+    SimpleRecord
+} from './dfx_generated/keywords/keywords.did';
+
+const keywords_canister = createActor('rrkah-fqaaa-aaaaa-aaaaq-cai', {
+    agentOptions: {
+        host: 'http://127.0.0.1:8000'
+    }
+});
+
+const tests: Test[] = [
+    {
+        name: 'deploy',
+        prep: async () => {
+            await new Promise((resolve) => setTimeout(resolve, 5000));
+
+            execSync(`dfx canister uninstall-code keywords || true`, {
+                stdio: 'inherit'
+            });
+
+            execSync(`dfx deploy keywords`, {
+                stdio: 'inherit'
+            });
+        }
+    },
+    {
+        name: 'keyword_variant',
+        test: async () => {
+            let keyword_variant: KeywordVariant = { raise: null };
+            const result = await keywords_canister.keyword_variant(
+                keyword_variant
+            );
+
+            return {
+                ok: 'raise' in result
+            };
+        }
+    },
+    {
+        name: 'simple_keyword',
+        test: async () => {
+            let simple_record: SimpleRecord = {
+                from: 'testing'
+            };
+            const result = await keywords_canister.simple_keyword(
+                simple_record
+            );
+
+            return {
+                ok: result.from === 'testing'
+            };
+        }
+    },
+    {
+        name: 'complex_keyword',
+        test: async () => {
+            const result = await keywords_canister.complex_keyword();
+
+            let expected_output: KeywordRecord = {
+                False: false,
+                True: 'False',
+                and: 1n,
+                assert: 2,
+                class: 3,
+                def: 4,
+                del: 5n,
+                elif: 6n,
+                except: 7,
+                finally: 8,
+                from: 9,
+                global: 10n,
+                import: 11.12,
+                is: 13.14,
+                lambda: ['False'],
+                nonlocal: 'False',
+                not: { from: 'False' },
+                or: {
+                    with_: false,
+                    with__: false,
+                    with___: false,
+                    with_______________________________________________________________________:
+                        false,
+                    with______1: false
+                },
+                pass: { raise: null },
+                raise: new Uint8Array([102, 97, 108, 115, 101]),
+                with: Principal.fromText('aaaaa-aa') as any,
```

Comment:
If you import Principal from `@dfinity/principal` (you might need to install it) then I think we can get rid of the cast to any